tidb-binlog/proto/go-binlog/bl.pb.go

```diff
@@ -1,11 +1,11 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
-// source: binlog.proto
+// source: bl.proto
```

Comment:
Name this to `slave_binlog.proto` or `secondary_binlog.proto`? `bl.proto` is quite strange.